redisinsight/api/src/modules/workbench/providers/workbench-commands.executor.ts

```diff
@@ -69,10 +69,12 @@ export class WorkbenchCommandsExecutor {
       nodeOptions,
       mode,
     } = dto;
-
-    const [command, ...commandArgs] = splitCliCommandLine(commandLine);
+    let command = commandLine;
 
     try {
+      const [parsedCommand, ...commandArgs] = splitCliCommandLine(commandLine);
+      command = parsedCommand;
+
```

Comment:
Did you move parsing command line into try catch block to handle error? Did you add additional handling for it? I'm not sure that I understand...

Reply:
Yep. I moved parsing command to try catch block, reason: do not throw Error when we can't parse command and args from command line.
For example: in pipeline or in group mode user sent
`set 1 1
json.set "key" . "{}`
we parsed and executed `set 1 1`, but cant parse and execute `json.set "key" . "{}`. In current realization we handle it as 500 error, but we executed `set 1 1` command

Reply:
With your solution we will return 1 succeed and 1 failed error with 200 OK. If it was a goal - I'm fine with this.

Reply:
However in this case it will be better to add error handling to return `{ status: fail, response: "some reasonable error"}` instead of InternalServerException